packages/jazz-tools/package.json

```diff
@@ -164,7 +162,6 @@
     "prosemirror-state": "^1.4.3",
     "prosemirror-transform": "^1.9.0",
     "react-native-fast-encoder": "^0.2.0",
```

Comment:
There are no usages of this dependency in our codebase, so I'm not sure why/when it is required. @gdorsi @Takeno do you happen to know?

Reply:
It has been introduced here: https://github.com/garden-co/jazz/commit/0d5ee3ed075dab7a37bcc22c0bbd01f151cb320c

/cc @boorad